ide-configuration/LICENSE-ASL-2.0-HEADER.txt

```diff
@@ -1,11 +1,11 @@
-Copyright $today.year Alfresco and/or its affiliates.
- 
+Copyright 2010-$today.year Alfresco Software, Ltd.
+
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
- 
-     http://www.apache.org/licenses/LICENSE-2.0
- 
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is this even used at all? Maybe other than IntelliJ browsers anyways would delete it if no good reason, duplication has maintenance burden.
If we keep it: `https` in this link instead of `http` so it's consistent with the rest

Reply:
not sure, looks like for other IDEs but then it's in the velocity syntax

Reply:
Okay, let's have keep it for now